source/gameengine/Rasterizer/RAS_OpenGLRasterizer/RAS_OpenGLRasterizer.cpp

```diff
@@ -932,6 +932,29 @@ void RAS_OpenGLRasterizer::DrawDebugCameraFrustum(SCA_IScene *scene, const MT_Ma
 		MT_Vector4(0.8f, 0.5f, 0.0f, 1.0f));
 }
 
+void RAS_OpenGLRasterizer::DrawDebugLightFrustum(SCA_IScene *scene, const MT_Matrix4x4& projmat, const MT_Matrix4x4& viewmat)
```

Comment:
If you don't use an inversed matrix in `KX_KetsjiEngine::DrawDebugLightFrustum` the inverse in `DrawDebugLightFrustum` became obsolete, isn't ?